src/services/lineaStateBridgeEventListener.js

```diff
@@ -0,0 +1,76 @@
+import { ethers } from "ethers";
+
+export const lineaStateBridgeEventListener = async ({
+  ethereumEndpoint,
+  lineaStateBridgeAddress,
+  l1MessageServiceAddress,
+}) => {
+  let provider;
+  let reconnectAttempts = 0;
+  const MAX_RECONNECT_ATTEMPTS = 10;
+  const RECONNECT_DELAY = 5000; // 5 seconds delay between reconnection attempts
+
+  const initializeProvider = () => {
+    provider = new ethers.providers.JsonRpcProvider(ethereumEndpoint);
```

Comment:
In the v6 version of Ethers providers are available in the root namespace:
```suggestion
    provider = new ethers.JsonRpcProvider(ethereumEndpoint);
```